addons/l10n_sk/demo/demo_company.xml

```diff
@@ -6,7 +6,7 @@
         <field name="street">Pařížská Street 25/31</field>
```

Comment:
Btw Pařížská Street is in Prague, not Bratislava :eyes: 
It doesn't change much, but maybe changing it to an actual street in Bratislava (and ideally one under the zip code added) would be better for consistency.